test/functional/revision_policies.js

```diff
@@ -314,5 +400,90 @@ describe('MVCC revision policy', function() {
     it('sets a TTL on all but the latest N entries (no 2ary indices)', function() {
         return revisionRetentionTest(this, 'revPolicyLatestTest-no2ary');
     });
-});
 
+    // Checks interval rev retention policy: need to ensure we have max 2 renders every 24 hours
+    // We add 3 renders on first day, 1 render on third day and 3 renders on the 4th day
+    // Renders number 1 (first one is never deleted), 3, 4, 5, 7 must survive, others should get removed
+    it('sets a TTL for interval rev policy', function() {
+        this.timeout(17000);
+        return createRenders(testIntervalSchema2, "Revisioned", 1000, [
+            // Day 1: 3 renders come
+            new Date("2015-04-01 12:00:00-0000"),
+            new Date("2015-04-01 12:10:00-0000"),
+            new Date("2015-04-01 12:50:00-0000"),
+            // Next day - nothing
+            // Next day - one revision comes
+            new Date("2015-04-03 12:00:00-0000"),
+            // Next day tree more
+            new Date("2015-04-04 12:00:00-0000"),
+            new Date("2015-04-04 12:30:00-0000"),
+            new Date("2015-04-04 13:00:00-0000")
+        ])
+        .delay(11000)
```

Comment:
@d00rman It's for TTLs to catch up. We could set a TTL smaller, though. Will do to have faster testing